### PR TITLE
TCD-145_improve-public-documentation

### PR DESCRIPTION
**Pull request by adebayoea**
[TCD-145
](https://liveintent.atlassian.net/browse/TCD-145)

[CM-848](https://liveintent.atlassian.net/browse/CM-848) Part 1 : Fix broken statuses 

The lgtm is no longer available, it will be removed and replace with CodeQL
Related PRs:
- https://github.com/LiveIntent/live-connect/pull/119

Author Todo List:

- [ ] Add/adjust tests (if applicable)
- [ ] Build in CI passes
- [ ] Latest master revision is merged into the branch
- [ ] Self-Review
- [ ] Set `Ready For Review` status

[CM-848]: https://liveintent.atlassian.net/browse/CM-848?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,23 +5,24 @@
...
-- Run full integration tests on multiple browsers (requires properly set Browserstack credentials) : `npm run test:it:browserstack`
+To quickly get started with the LiveConnect module, perform the following steps via the command line interface.
+
+- Installation: Begin by installing the module using npm. Run the following command: `npm install`.
+- Code Formatting: Ensure your code adheres to the provided eslint file for consistent formatting. Use the command `npm run fix-js` to automatically format your code.
+- Unit Testing: Validate the functionality of your code by running the unit tests. Execute the command `npm run test:unit` to initiate the tests.
+- Integration Testing: Verify the integration of LiveConnect by running the integration tests against a dockerized Chrome browser. Use the command `npm run test:it:docker:chrome` to perform these tests.
+- Cross-Browser Testing: For a comprehensive evaluation, conduct full integration tests on multiple browsers. Please ensure you have valid Browserstack credentials configured. Run the command `npm run test:it:browserstack` to execute the tests across various browsers.
```
Comment (SuperIzya) [inline README.md:22]: IMHO, it is better if you mention that browserstack needs credentials.

**Review by SuperIzya — APPROVED** [review]

**Review by SuperIzya — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -5,23 +5,24 @@
 [![codecov](https://codecov.io/gh/liveintent-berlin/live-connect/branch/master/graph/badge.svg?token=P5sRpM4U6k)](https://codecov.io/gh/liveintent-berlin/live-connect)
 [![Total alerts](https://img.shields.io/lgtm/alerts/g/LiveIntent/live-connect.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/LiveIntent/live-connect/alerts/)
 [![Language grade: JavaScript](https://img.shields.io/lgtm/grade/javascript/g/LiveIntent/live-connect.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/LiveIntent/live-connect/context:javascript)
-[![devDependencies Status](https://david-dm.org/LiveIntent/live-connect/dev-status.svg)](https://david-dm.org/LiveIntent/live-connect?type=dev)
-[![dependencies Status](https://david-dm.org/LiveIntent/live-connect/status.svg)](https://david-dm.org/LiveIntent/live-connect)
+[//]: # "[![devDependencies Status](https://david-dm.org/LiveIntent/live-connect/dev-status.svg)](https://david-dm.org/LiveIntent/live-connect?type=dev)"
```
Comment (3link) [inline README.md:8]: What are these changes for? This is what I see when I view the page:
Screenshot 2023-06-01 at 17 12 17

File: README.md [hunk 3]
```diff
@@ -5,23 +5,24 @@
...
+[//]: # "[![dependencies Status](https://david-dm.org/LiveIntent/live-connect/status.svg)](https://david-dm.org/LiveIntent/live-connect)"
 
-# Main concepts
-LiveConnect module is used to create, and or gather first party identifiers of your choosing, and sending that information to a defined endpoint which is responsible for gathering and processing that data.
-What LiveConnect provides is a simple interface to collect the identifiers from a page, and collect user interactions along with those identifiers.
-To see what kind of data is being sent, check [what is being sent](#what-is-being-sent).
+## Main concepts
+The LiveConnect module offers a convenient solution for generating and collecting first-party identifiers based on your preferences. It enables you to send this information to a designated endpoint, where it can be efficiently processed and analyzed. With LiveConnect, you gain a straightforward interface that facilitates the collection of identifiers from web pages, as well as capturing user interactions alongside these identifiers.
```
Comment (3link) [inline README.md:12]: The data can be sent to any endpoint. Also ones that are not owned by LiveIntent. I would rather change the description to somethings like 
```suggestion
The LiveConnect module offers a convenient solution for generating and collecting first-party identifiers based on your preferences, and sending this information to a designated endpoint. With LiveConnect, you gain a straightforward interface that facilitates the collection of identifiers from web pages, as well as capturing user interactions alongside these identifiers.
```

File: README.md [hunk 4]
```diff
@@ -5,23 +5,24 @@
...
 
 ## Quick start
-- With [npm](https://www.npmjs.com/): `npm install`
-- Format the code according to the eslint file: `npm run fix-js`
-- Run unit tests: `npm run test:unit`
-- Run integration tests against a dockerized chrome: `npm run test:it:docker:chrome`
-- Run full integration tests on multiple browsers (requires properly set Browserstack credentials) : `npm run test:it:browserstack`
+To quickly get started with the LiveConnect module, perform the following steps via the command line interface.
+
+- Installation: Begin by installing the module using npm. Run the following command: `npm install`.
```
Comment (3link) [inline README.md:18]: Strictly speaking, `npm install` installs the dependencies, not the module.

**Review by 3link — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (adebayoea) [reply README.md]: I've now changed that subsection to - Dependency installation. Is that sufficient or
should there be specific details about LiveConnect installation? If yes, how does that work?

File: README.md [hunk 5]
```diff
@@ -5,23 +5,24 @@
...
+- Code Formatting: Ensure your code adheres to the provided eslint file for consistent formatting. Use the command `npm run fix-js` to automatically format your code.
+- Unit Testing: Validate the functionality of your code by running the unit tests. Execute the command `npm run test:unit` to initiate the tests.
+- Integration Testing: Verify the integration of LiveConnect by running the integration tests against a dockerized Chrome browser. Use the command `npm run test:it:docker:chrome` to perform these tests.
```
Comment (SuperIzya) [inline README.md:21]: Before one can run integration tests locally, one needs to adjust some test domains (shell for Linux):
```
echo "127.0.0.1 bln.test.liveintent.com" | sudo tee -a /etc/hosts
echo "127.0.0.1 test.liveintent.com" | sudo tee -a /etc/hosts
echo "127.0.0.1 me.idex.com" | sudo tee -a /etc/hosts
echo "127.0.0.1 schmoogle.com" | sudo tee -a /etc/hosts
echo "127.0.0.1 framed.test.liveintent.com" | sudo tee -a /etc/hosts
echo "127.0.0.1 double-framed.test.liveintent.com" | sudo tee -a /etc/hosts
echo "127.0.0.1 baked.liveintent.com" | sudo tee -a /etc/hosts
```

Comment (SuperIzya) [reply README.md]: These domains are crucial if you run any of `npm run test:it:*`. This domain adjustment is provided in test section, but it's outdated there and incomplete

File: README.md (hunk 2 above)
Comment (adebayoea) [reply README.md]: The [CM-484 ticket](https://liveintent.atlassian.net/browse/CM-848) is for taking care of the  issues:

**Review by 3link — APPROVED** [review]

**Pull request merged**